### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by LuBuss**
implement dockerfile

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		nano \
+		ca-certificates
+
+# install PM2
+RUN npm install pm2 -g
+
+RUN mkdir -p /var/www \ 
+	&& cd /var/www \ 
+	&& mkdir -p cezerin2 
+
```
Comment (ngoyal16) [inline Dockerfile:27]: can add cezerin2 directly in single command 

after creating directly make workdir before running commands this way DOCKER FILE can be reduced.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# script to run Nginx and PM2
+COPY docker-entrypoint.sh /usr/local/bin/
+RUN chmod +x "/usr/local/bin/docker-entrypoint.sh"
+
+# build project
+RUN cd /var/www/cezerin2 \
+    && npm i
+
+WORKDIR /var/www/cezerin2
+
```
Comment (ngoyal16) [inline Dockerfile:44]: expose the listen port also add image optimizer to Nginx.

**Review by ngoyal16 — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (LuBuss) [reply Dockerfile:44]: don't feel nginx should be running here. I feel this should be happening on store and admin

Comment (ngoyal16) [reply Dockerfile:44]: we need nginx here for image resizing part.

**Pull request closed without merging**